### ISSUE TITLE
Multiorg generation command uses `singleorg`

### ISSUE THREAD

**Issue by Alfetta159** [Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2#4]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [ x] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
>
run multiorg dotnet CLI command to generate project.

### Any log messages given by the failure
>
Creates wrong appsettings.json

### Expected/desired behavior
>
Should use `multiorg`.

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)
N/A
### Versions
>

### Mention any other details that might be useful

Line #50 of ReadMe file

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by Rick-Anderson** [comment]
@Alfetta159  what is the exact command you ran? Can you provide a link to the *ReadMe* file?

**Comment by jmprieur** [comment]
@Alfetta159 
Actually, this does not matter as anyway Dotnet new creates an AAD V1 project (for Work & School accounts only), whereas this sample is about AAD V2 (supporting Work and School and Microsoft personal accounts). I tried both templates and found that the SingleOrg one was a better starting point for be changed to support the AAD V2 endpoint.

I'll be a bit more explicit in the README about that.
cc: @Rick-Anderson 

The right "fix" is to provide AAD V2 ASP.NET Core templates. I'll start a conversation with the ASP.NET Core team about that.

**Comment by jmprieur** [comment]
Closing as fixed by commit #8bc83219e0aa366dda8529fe2a6183e40afdc523
